### DISCUSSION TITLE
Image with CTkLabel?

### DISCUSSION THREAD

**Discussion by 32-git** [TomSchimansky/CustomTkinter#954] [Q&A]
After updating to the latest version (5.0.3) I have realised that the image I used to have set as a CTkLabel is no longer appearing (added via attribute "image="). 

Upon inspection into the wiki, I noticed this is no longer an attribute of CTkLabel and therefore would like to kindly ask whether there would be any _new_ way to achieve the same.

**Comment by 32-git** [comment]
I've realised that it is possible to make the image a CTkImage first and then a button with that image. Accordingly, in my case, the button would have no command or callback function.

However, if I may, I would still like to ask whether an image attribute for the label is planned to be re-introduced.

**Reply by TomSchimansky** [reply]
You can use a CTkImage or a PIL.PhotoImage for the CTkLabel, both should work.
